armi/utils/dochelpers.py

```diff
@@ -68,6 +68,49 @@ def create_table(rst_table, caption=None, align=None, widths=None, width=None):
     return "\n".join(rst)
 
 
+def createListTable(rows, caption=None, align=None, widths=None, width=None):
+    """Take a list of data, and produce an RST-type string for a list-table.
+
+    Parameters
+    ----------
+    rows: list
+        List of input data (first row is the header).
+    align: str
+        “left”, “center”, or “right”
```

Comment:
shouldn't those quotes be `"` rather than the weirder unicode chars?

Reply:
Done